### PR TITLE
feat(supernova): remove auth and pass initial filters per appProp

### LINKED ISSUE #422: refactor(supernova): remove all auth related code
Remove auth store slice and other occurencies.

### PR DESCRIPTION
**Pull request by TilmanHaupt**
# Summary

This PR removes the authentication integration from the application, simplifying the access process and allowing a full preview of the application within a PR. Additionally, it introduces a new appProp called initialFilters and username, which allows default filters and auditing to be applied when initializing the application. 

# Related Issues

#368 
closes #422 

# Testing Instructions

1. `npm i`
2. `npm run TASK`

# Checklist

- [x] I have performed a self-review of my code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.
- [ ] I have made corresponding changes to the documentation (if applicable).
- [x] My changes generate no new warnings or errors.

### CODE REVIEW

**Comment by andypf** [comment]
Will it work if I approve this PR, or should we wait for the other PRs to be approved first?

**Review by barsukov — APPROVED** [review]
LGTM

**Review by ArtieReus — CHANGES_REQUESTED** [review]

File: apps/supernova/secretProps_sample.js [hunk 1]
```diff
@@ -5,12 +5,16 @@ module.exports = {
   endpoint: "",
   // These are the labels that are used to filter alerts in the UI, e.g. ["app", "severity", "status"]
   filterLabels: [],
+  // initial filters that are applied when the app is loaded, see README.md for more information
+  initialFilters: [],
```
Comment (ArtieReus) [inline apps/supernova/secretProps_sample.js:9]:
```suggestion
  initialFilters: {},
```

**Review by ArtieReus — CHANGES_REQUESTED** [review]

**Review by ArtieReus — APPROVED** [review]
lgtm

**Review by hodanoori — APPROVED** [review]
lgtm

File: apps/supernova/src/lib/createFiltersSlice.js [hunk 2]
```diff
@@ -25,8 +25,26 @@ const parsePredefinedFilters = (predefinedFilters) => {
...
+
+  if (typeof initialFilters !== "object") {
+    console.warn("[supernova]::parseInitialFilters: initialFilters object is not an object")
+    return {}
+  }
+
+  // Check if all keys are in filterLabelValues
+  if (!Object.keys(initialFilters).every((key) => filterLabels.includes(key))) {
+    console.warn("[supernova]::parseInitialFilters: Some keys of the initialFilters object are not in the labels")
+    return {}
```
Comment (edda) [inline apps/supernova/src/lib/createFiltersSlice.js:39]: I would not discard the initialFilters object in this case. Instead do display the warning but filter the object and keep the entries that are ok.

File: apps/supernova/src/lib/createFiltersSlice.js [hunk 3]
```diff
@@ -25,8 +25,26 @@ const parsePredefinedFilters = (predefinedFilters) => {
...
+  if (!initialFilters) return initialFiltersState.initialFilters
+
+  if (typeof initialFilters !== "object") {
+    console.warn("[supernova]::parseInitialFilters: initialFilters object is not an object")
+    return {}
+  }
+
+  // Check if all keys are in filterLabelValues
+  if (!Object.keys(initialFilters).every((key) => filterLabels.includes(key))) {
+    console.warn("[supernova]::parseInitialFilters: Some keys of the initialFilters object are not in the labels")
```
Comment (edda) [inline apps/supernova/src/lib/createFiltersSlice.js:38]: Change message to:
"Some keys of the initialFilters object are not valid filter labels. They must be configured as filterLabels first."

**Pull request merged**